Geometry/GEMGeometryBuilder/src/ME0GeometryBuilder.cc

```diff
@@ -0,0 +1,523 @@
+/*
+//\class ME0GeometryBuilder
+
+ Description: ME0 Geometry builder from DD & DD4hep
+              DD4hep part added to the original old file (DD version) made by M. Maggi (INFN Bari)
+//
+// Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osborne made for DTs (DD4HEP migration)
+//          Created:  29 Apr 2019 
+*/
+#include "Geometry/GEMGeometryBuilder/src/ME0GeometryBuilder.h"
+#include "Geometry/GEMGeometry/interface/ME0Geometry.h"
+#include "Geometry/GEMGeometry/interface/ME0EtaPartitionSpecs.h"
+
+#include "DetectorDescription/Core/interface/DDFilter.h"
+#include "DetectorDescription/Core/interface/DDFilteredView.h"
+#include "DetectorDescription/Core/interface/DDSolid.h"
+
+#include "Geometry/MuonNumbering/interface/MuonDDDNumbering.h"
+#include "Geometry/MuonNumbering/interface/MuonBaseNumber.h"
+#include "Geometry/MuonNumbering/interface/ME0NumberingScheme.h"
+
+#include "DataFormats/GeometrySurface/interface/TrapezoidalPlaneBounds.h"
+
+#include "DataFormats/GeometryVector/interface/Basic3DVector.h"
+
+#include "DataFormats/Math/interface/CMSUnits.h"
```

Comment:
Delete CMSUnits.h. I accidentally mentioned it in a comment, but I thought I had deleted that comment.

Reply:
@cvuosalo : done